### DISCUSSION TITLE
GoPro 11 Metadata: TZON

### DISCUSSION THREAD

**Discussion by ghost** [gopro/labs#774] [Q&A] [answered]
Hey there, 

can someone tell me for what TZON in the metadata stands for?

For some reasons after the latest GP 11 labs update, the picture of the cam seems kinda more noisy then i'm used too.
Since the only difference in my metadata compared to previous recordings is TZON, i thought it might be related to that?

Thanks in advance

**Comment by dnewman-gpsw** [comment, marked as answer]
Time Zone, so not related.  

The only Labs controls that directly changes the noise look is !MNR01=x  (1 is the most noise, 0 the least, 50 = 50%.)  Of course increasing ISO also increases noise.

**Reply by ghost** [reply]
Thanks for the clarification.
